pkg/models/manifest.go

```diff
@@ -224,6 +225,24 @@ func ApplicationNameTooLong(name string) bool {
 	return len(name) > MaxKubernetesResourceNameLength
```

Comment:
Could we get rid of this function? It seems it won't be used anymore

Reply:
Great question.  It is still in use by hashNameIfTooLong() so I left it in.